### DISCUSSION TITLE
z/Open Editor folder and file icons - question

### DISCUSSION THREAD

**Discussion by davidkjackson54** [IBM/zopeneditor-about#140] [Q&A]
Is z/Open Editor using its own set of icons for file and folders  - or is it using the already installed 'Minimal' File Icon Theme that is installed as part of VSCode - even though it isn't the default File Icon Theme in use.  "Seti" is the default File Icon Theme.

**Comment by phaumer** [comment]
We are not setting any preferences for icons or install our own. It should be using whatever the user has configured.

**Comment by davidkjackson54** [comment]
Hi Peter,
VSCode comes with 2 predefined File ICON themes - 'minimal' and 'seti'. 
By default seti is used . There is no 'folder' icon availble in seti 
So a tree looks like this:
![image](https://user-images.githubusercontent.com/46171060/124977399-47339600-dfe5-11eb-9b2a-5d7f3ea6c282.png)

However if I switch to the 'minimal' File icon theme it looks like this:
![image](https://user-images.githubusercontent.com/46171060/124977547-734f1700-dfe5-11eb-80a6-3a597f99427a.png)
Which looks remarkably like the icons being used by z/Open Editor.
![image](https://user-images.githubusercontent.com/46171060/124978171-46e7ca80-dfe6-11eb-954d-0f6cdd7142ef.png)

I then installed a completely new icon theme - 'Material file Icon Theme' and set it as default 
z/Open Editor did not change - yet my own VSCODE Explorer does change.
![image](https://user-images.githubusercontent.com/46171060/124978744-063c8100-dfe7-11eb-983d-2577f6954f92.png)

So it seems on the face of it as though z/Open Editor is somehow using the Minimal Icon Theme even when not the default. 
That is fine if that is so, but I'd sure like to know how you have accomplished that!

**Comment by phaumer** [comment]
Oh, @davidkjackson54, the screen shots you are showing are from Zowe Explorer. That extension does provide and reference specific icon sets. I have not worked on that, so it might be better to ask here: <https://github.com/zowe/vscode-extension-for-zowe/discussions>
